src/components/contacts/ContactList.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react'
+import ContactItem from './ContactItem'
+import contactsJson from '../../contacts.json'
+
+
+
+export default function ContactList() {
+
+  const [contacts, setContacts] = useState([])
+  const [remainContacts, setRemainContacts] = useState([])
+
+  useEffect(() => {
+    setContacts(contactsJson.slice(0, 5))
+    setRemainContacts(contactsJson.slice(5))
+  }, [])
+
+
+  const handleAddClick = () => {
+    if (remainContacts.length !== 0) {
```

Comment:
```suggestion
   const remainContacts = contactsJson.filter(contact => !contacts.includes(contact))
    if (remainContacts.length !== 0) {
```
Aqui basicamente nos quedamos con los que sabemos que no estan en el estado (niego el includes porque quiero los que no esten, si estan todos el filter devolvera un array vacio!!)